eng/Packages.Data.props

```diff
@@ -128,7 +128,7 @@
     <PackageReference Update="OpenTelemetry.Instrumentation.AspNetCore" Version="1.5.1-beta.1" />
     <PackageReference Update="OpenTelemetry.Instrumentation.Http" Version="1.5.1-beta.1" />
     <PackageReference Update="OpenTelemetry.Instrumentation.SqlClient" Version="1.5.1-beta.1" />
-    <PackageReference Update="OpenTelemetry.PersistentStorage.FileSystem" Version="1.0.0-beta.2" />
+    <PackageReference Update="OpenTelemetry.PersistentStorage.FileSystem" Version="1.0.0" />
     <PackageReference Update="OpenTelemetry.ResourceDetectors.Azure" Version="1.0.0-beta.2" />
```

Comment:
Beta component :(

Reply:
Are you commenting about the ResourceDetector?
I don't think we have a plan to stabilize this anytime soon.